tests/console/rmt/rmt_feature.pm

```diff
@@ -18,10 +18,12 @@ use utils;
 
 sub run {
     select_console 'root-console';
+    record_info('rmt server setup');
```

Comment:
If you don't enter the content of the record, click on it will show a null dialog. I think you'd better add the content for it, title is ok if you have nothing else to describe it. Same for other record_infos.

Reply:
OK, done, please help to review, thanks.